### PR TITLE
Tests for resetting stats

### PR DESCRIPTION
**Pull request by vartanbeno**
Instrumented test. Issue will be update to include the once this PR is merged to master.

### CODE REVIEW

**Comment by vartanbeno** [comment]
Merge conflicts resolved

**Review by OlivierTD — CHANGES_REQUESTED** [review]

File: app/src/test/java/resetStatsTest/ResetStatsTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        StatisticsFragment statisticsFragment = new StatisticsFragment();
+        Button mockButton = Mockito.mock(Button.class);
+
+        statisticsFragment.setResetStatsButton(mockButton);
+        statisticsFragment.setResetStatsFunctionality();
+
+        // test that an arbitrary OnClickListener object is set on the button
+        verify(mockButton).setOnClickListener((View.OnClickListener) any());
+
+        assertEquals(statisticsFragment.getResetStatsButton(), mockButton);
```
[the remark this answers is not in the document]
Comment (vartanbeno) [reply app/src/test/java/resetStatsTest/ResetStatsTest.java]: @OlivierTD After much consideration I believe there's no functionality that can be unit tested. I decided to take it out.

**Review by anania-y — APPROVED** [review]
Tests look good. Run well on my end.

**Comment by SaiShan** [comment]
Tests are passing on my device too. Code which included the changes requested by @OlivierTD were removed.

Please wait for CircleCi to be done before merging.

**Review by SaiShan — APPROVED** [review]

**Pull request merged**